alfresco-search/src/test/java/org/alfresco/solr/query/afts/qparser/QParserPluginTest.java

```diff
@@ -321,7 +322,8 @@ public void wildcards()
         assertAQuery("TEXT:\"*esco\"", 1);
         assertAQuery("TEXT:\"*sco\"", 1);
         assertAQuery("TEXT:\"*co\"", 1);
-        assertAQuery("TEXT:\"*o\"", 1);
+        // it finds cm:name:two and file fourteen because of content
```

Comment:
Nice effort to try to salvage this test.  Potentially we could improve things further by including information about the expected results in the assertion message, but that's probably outside the scope of this task.